### DISCUSSION TITLE
Termux-play-store running on an armeabi-v7a - "permission denied" error

### DISCUSSION THREAD

**Discussion by tonialpin** [termux-play-store/termux-issues#263] [Q&A]
Hi everyone,

I'm currently working on getting termux-play-store running on an armeabi-v7a Android device.

I've already made some progress:
- Adjusted the settings and reduced the SDK version.
- Successfully installed the termux-play-store app on the device!
- Managed to complete the bootstrap installation by including the bootstrap-arm file.

However, I'm now encountering an issue. When it tries to execute /files/usr/bin/bash in the /usr/bin/login file, I get a "permission denied" error. It seems that the bootstrap-arm.zip might not be compatible with termux-play-store.

Does anyone have any insights or suggestions on how to resolve this?

I’d really appreciate any help or advice. Thanks in advance!

**Comment by tonialpin** [comment]
Hi @fornwall 
any idea on this? Could you support here :) ?

**Reply by robertkirkman** [reply]
my suggestion: try redoing the same steps you did, but **without** modifying the SDK version. 

instead, test the 32-bit `.apk` you build on one or more of the following:

- a real bare metal 32-bit android 11+ device
- a bare metal 64-bit android 11+ device that has [traditional low-level multilib support enabled, allowing it to run 32-bit apps including 32-bit builds of Termux](https://source.android.com/docs/setup/create/64-bit-builds) 
  - (i.e. any android 11+ device that is _not_ [like this](https://android-developers.googleblog.com/2022/10/64-bit-only-devices.html))
- an android 11+ true full system ARM docker-inside-emulator of the above system, like my [redroid configuration](https://gist.github.com/robertkirkman/f79441c79811ad263f2f881f7864e793?permalink_comment_id=5103748#gistcomment-5103748) 
  - ([**EXTREMELY SLOW**](https://github.com/remote-android/redroid-doc/issues/705))

Overall, the idea is that basically you need to stop reducing the API level, in order to use Google Play Termux, because Google Play Termux is for Android 11+.

**Reply by tonialpin** [reply]
Thanks Robert for your reply. Unfortunately, my target device is running on android 10...

**Reply by robertkirkman** [reply]
Can what you need to do be achieved using [F-Droid Termux](https://f-droid.org/en/packages/com.termux/)?
F-Droid Termux is for Android 10. It is also available precompiled for 32-bit processors already.
